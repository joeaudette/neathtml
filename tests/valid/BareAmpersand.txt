<a href="http://mywebsite.com/GalleryImageEdit.aspx?mid=9&pageindex=3&pageid=6">http://mywebsite.com/GalleryImageEdit.aspx?mid=9&pageindex=3&pageid=6</a>.
