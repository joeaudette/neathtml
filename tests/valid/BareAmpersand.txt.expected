<a href="http://mywebsite.com/GalleryImageEdit.aspx?mid=9&amp;pageindex=3&amp;pageid=6">http://mywebsite.com/GalleryImageEdit.aspx?mid=9&amp;pageindex=3&amp;pageid=6</a>.
